src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -115,11 +115,11 @@ public static function getStyleDefinition($name)
             self::$styles = self::initStyles();
         }
 
-        if (!self::$styles[$name]) {
-            throw new InvalidArgumentException(sprintf('Style "%s" is not defined.', $name));
+        if (isset(self::$styles[$name])) {
+            return self::$styles[$name];
```

Comment:
nope, that wouldn't make sense, but if one already has the TableStyle object, returning it unchanged as a side effect (undocumented in the docblock) would worry me either...